apps/tailwind-components/components/input/TextArea.vue

```diff
@@ -52,7 +52,7 @@ function onInput(event: Event) {
     :required="required"
     :placeholder="placeholder"
     :disabled="disabled"
-    class="w-full pr-16 font-sans text-black text-gray-300 outline-none rounded-textarea-input h-60 pl-3 shadow-search-input focus:shadow-search-input hover:shadow-search-input search-input-mobile border py-2"
+    class="w-full pr-16 font-sans text-black text-gray-300 h-[112px] outline-none rounded-textarea-input pl-3 shadow-search-input focus:shadow-search-input hover:shadow-search-input search-input-mobile border py-2"
```

Comment:
Should we use the `rows` attribute here instead of setting height? 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#rows